.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup LXD
+        uses: whywaita/setup-lxd@v1
```

Comment:
Use `canonical/setup-lxd` here, and you can remove manual iptables tweaking in rockcraft section.